grammar/BrightScript.g4

```diff
@@ -558,6 +568,32 @@ WS
     : [ \t]+ -> skip
     ;
 
+CONDITIONAL_CONST
+    : '#' C O N S T
+    ;
+
+CONDITIONAL_ELSE
+    : '#' ELSE
+    ;
+
+CONDTIONAL_ELSEIF
```

Comment:
typo spelling here

should be CONDITIONAL_ELSEIF

Reply:
Some day I'll make my 8th grade typing teacher proud. Today is not that day. 😃